keycloak/aam-theme-codo/email/theme.properties

```
@@ -0,0 +1,2 @@
+parent=aam-theme
+locales=en,de
```

```suggestion
locales=de
```

Falls erlaubt, nur "de". Habe das englische Template gar nicht hinzugefügt hier.